docs/pipelines/multisample/shiva.md

```diff
@@ -97,15 +100,18 @@ At this moment the following variant callers can be used
 | varscan_cns_singlesample | <a href="http://varscan.sourceforge.net/">varscan</a> |  |
 
 ##### Somatic
+
+In contrast to germline variantcalling does somatic variantcalling does not have a direct assumption about the number of alleles. Some can also take a control into account, like MuTect2. Having a control is useful when analysing tumor samples.
```

Comment:
`In contrast to germline variantcalling does somatic variantcalling` -> `In contrast to germline variant calling, somatic variant calling`